llama_stack/apis/telemetry/telemetry.py

```diff
@@ -130,3 +144,8 @@ async def log_event(self, event: Event) -> None: ...
 
     @webmethod(route="/telemetry/get-trace", method="GET")
     async def get_trace(self, trace_id: str) -> Trace: ...
+
+    @webmethod(route="/telemetry/get-traces-for-session", method="POST")
+    async def get_traces_for_session(
+        self, session_id: str, lookback: str = "1h", limit: int = 100
```

Comment:
Should we make `lookback` window to be int in second or smt? What does the `limit` mean?

Reply:
1. The argument for lookback is actually a string for jaeger as well. I think its easier to specify the window using 1h or 5s instead of int.
2. If its not part of a session, we should be able to get the trace using just the trace id. We already have a endpoint for that, but not implemented for open telemetry. I can add that.